build.gradle

```diff
@@ -196,20 +196,20 @@ allprojects {
 
   dependencies {
     implementation files(
-      'consumers/build/libs/civil-sdt-consumers-0.0.1-SNAPSHOT.jar',
-      'dao/build/libs/civil-sdt-dao-0.0.1-SNAPSHOT.jar',
-      'dao-api/build/libs/civil-sdt-dao-api-0.0.1-SNAPSHOT.jar',
-      'domain/build/libs/civil-sdt-domain-0.0.1-SNAPSHOT.jar',
-      'handlers/build/libs/civil-sdt-handlers-0.0.1-SNAPSHOT.jar',
-      'handlers-api/build/libs/civil-sdt-handlers-api-0.0.1-SNAPSHOT.jar',
-      'interceptors/build/libs/civil-sdt-interceptors-0.0.1-SNAPSHOT.jar',
-      'producers/build/libs/civil-sdt-producers-0.0.1-SNAPSHOT.jar',
-      'producers-api/build/libs/civil-sdt-producers-api-0.0.1-SNAPSHOT.jar',
-      'services/build/libs/civil-sdt-services-0.0.1-SNAPSHOT.jar',
-      'services-api/build/libs/civil-sdt-services-api-0.0.1-SNAPSHOT.jar',
-      'transformers/build/libs/civil-sdt-transformers-0.0.1-SNAPSHOT.jar',
-      'utils/build/libs/civil-sdt-utils-0.0.1-SNAPSHOT.jar',
-      'validators/build/libs/civil-sdt-validators-0.0.1-SNAPSHOT.jar')
+      "consumers/build/libs/civil-sdt-consumers-${version}.jar",
+      "dao/build/libs/civil-sdt-dao-${version}.jar",
+      "dao-api/build/libs/civil-sdt-dao-api-${version}.jar",
+      "domain/build/libs/civil-sdt-domain-${version}.jar",
+      "handlers/build/libs/civil-sdt-handlers-${version}.jar",
```

Comment:
why this change is here

Reply:
I think these were originally part of #28 then subsequently changed.

Reply:
Either caused from timing of the branch creation (or merges since)